### PR TITLE
bug fix for private-active-active test

### PR DESCRIPTION
**Pull request by anniehedgpeth**
## Background

This PR was opened due to failed / flaky tests in the `private-active-active` test module.

Relates #183

## This PR makes me feel

just work

### CODE REVIEW

File: tests/private-active-active/templates/cloud-config-proxy.yaml [hunk 1]
```diff
@@ -37,7 +37,6 @@ write_files:
             refresh_pattern .       0   20% 4320
 
 packages:
-    - https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
     - squid
 
 runcmd:
```
Comment (aaron-lane) [inline tests/private-active-active/templates/cloud-config-proxy.yaml:42]: Based on our internal version of this setup, we should be able to replace all of these commands with just `service squid reload` 👇🏼

File: tests/private-tcp-active-active/proxy.tf [hunk 2]
```diff
@@ -20,17 +20,6 @@ resource "aws_security_group_rule" "proxy_ingress_mitmproxy" {
   security_group_id = aws_security_group.proxy.id
 }
 
-resource "aws_security_group_rule" "proxy_ingress_ssh" {
```
Comment (anniehedgpeth) [inline tests/private-tcp-active-active/proxy.tf:23]: [This test](https://github.com/hashicorp/terraform-aws-terraform-enterprise/runs/2843576320?check_suite_focus=true) passed when this rule was removed.

File: tests/private-active-active/proxy.tf [hunk 3]
```diff
@@ -25,7 +26,14 @@ resource "aws_instance" "proxy" {
     aws_security_group.proxy.id,
   ]
 
-  user_data = data.template_cloudinit_config.config_proxy.rendered
+  user_data = base64encode(
+    templatefile(
+      "${path.module}/templates/squidproxy.sh.tpl",
```
Comment (anniehedgpeth) [inline tests/private-active-active/proxy.tf:31]: [This run](https://github.com/hashicorp/terraform-aws-terraform-enterprise/runs/2850073122?check_suite_focus=true) passed when I switched from cloud-init to a bash script.

**Review by aaron-lane — APPROVED** [review]
We'll get you some day, Private Active/Active. 👊🏼 

LGTM 🧪

**Pull request merged**